source/manuals/programming-languages/editorconfig

```diff
@@ -1,15 +1,50 @@
-[*.java]
+# http://EditorConfig.org
+# https://gds-way.cloudapps.digital/manuals/programming-languages/editorconfig
+# Copy into project root and rename to .editorconfig
+
+# uncomment if top-most EditorConfig file
+# root = true
+
+[*]
 indent_style: space
-indent_size: 4
-end_of_line: lf
-charset: utf-8
+charset = utf-8
+end_of_line = lf
 insert_final_newline: true
+trim_trailing_whitespace = true
+
+[*.java]
+indent_size: 4
 continuation_indent_size: 8
+/* this wasn't explicitly set, so setting to false until told otherwise */
+trim_trailing_whitespace = false
```

Comment:
This wasn’t set because the GDS Way Java style guide doesn’t say anything about it: https://github.com/alphagov/gds-way/pull/43#issuecomment-323073119

Reply:
Thanks for the update @alexbishop1, I'll read through those comments.